Sources/LicenseList/Library.swift

```diff
@@ -12,3 +12,27 @@ public struct Library: Hashable {
     public let licenseType: String
     public let licenseBody: String
 }
+
+extension Library {
+    static public var libraries: [Library] {
+        var libraries: [Library] = []
+        guard let fileURL = Bundle.main.url(forResource: "license-list", withExtension: "plist"),
+              let data = try? Data(contentsOf: fileURL),
+              let plist = try? PropertyListSerialization.propertyList(from: data, format: nil),
+              let dict = plist as? [String: Any] else {
+            return libraries
+        }
+        let _libraries = (dict["libraries"] as? [[String: Any]])?.compactMap({ info -> Library? in
+            guard let name = info["name"] as? String,
+                  let type = info["licenseType"] as? String,
+                  let body = info["licenseBody"] as? String else {
+                return nil
+            }
+            return Library(name: name,
+                           licenseType: type,
+                           licenseBody: body)
+        })
+        guard let libraries = _libraries else { return libraries }
+        return libraries
```

Comment:
LibraryからlicenseTypeを除いたため、以下のように修正してください。
Since licenseType has been removed from Library, please modify as follows.

```swift
return (dict["libraries"] as? [[String: Any]])?.compactMap({ info -> Library? in
    guard let name = info["name"] as? String,
          let type = info["licenseType"] as? String else {
        return nil
    }
    return Library(name: name, licenseBody: body)
}) ?? []
```

Reply:
`licenseType` -> `licenseBody`でした。

Reply:
Fixed in [74086e9](https://github.com/cybozu/LicenseList/pull/8/commits/74086e95aa016d3ad4fb39219bd5d9332432feb1) & [d423e65](https://github.com/cybozu/LicenseList/pull/8/commits/d423e6552ed0e2a88a5fee15ed24f479e3e5eb9f)